src/GraphQL/Federation/Extensions/FederationGraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,48 @@
+using GraphQL.DI;
+using GraphQL.Federation.Enums;
+using GraphQL.Federation.Types;
+using GraphQL.Utilities;
+
+namespace GraphQL.Federation.Extensions;
+
+/// <summary>
+/// Federation extensions for IGraphQLBuilder
```

Comment:
Please, use `<see cref` constructs when referencing types/methods.

Reply:
> ends tests with dot

Meaning: End xml comments with periods, such as the following:

```suggestion
/// Federation extensions for <see cref="IGraphQLBuilder" />.
```